racecar:

  left_rear_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: left_rear_axle
    pid: {p: 20.0, i: 0.10, d: 0.0}


  right_rear_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: right_rear_axle 
    pid: {p: 20.0, i: 0.10, d: 0.0}


  left_front_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: left_front_axle
    pid: {p: 20.0, i: 0.10, d: 0.0}


  right_front_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: right_front_axle
    pid: {p: 20.0, i: 0.10, d: 0.0}
    
  left_steering_hinge_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_steering_joint
    pid: {p: 100.0, i: 0.1, d: 1.0}
  
  right_steering_hinge_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_steering_joint
    pid: {p: 100.0, i: 0.1, d: 1.0}


  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
